docs/website/docusaurus.config.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const theme = require('prism-react-renderer/themes/github');
+
+module.exports = {
+  title: 'Renovate Docs',
+  tagline: 'Documentation for Renovate',
+  url: 'https://docs.renovate.com',
+  baseUrl: '/',
+  onBrokenLinks: 'warn',
+  onDuplicateRoutes: 'throw',
+  favicon: 'img/logo.png',
+  themeConfig: {
+    colorMode: {
+      disableSwitch: true,
+      respectPrefersColorScheme: true,
+    },
+    navbar: {
+      hideOnScroll: true,
+      title: 'Renovate Docs',
+      logo: {
+        alt: 'Renovate Logo',
+        src: 'img/logo.png',
+      },
+      items: [
+        {
+          href: 'https://github.com/renovatebot/renovate',
+          label: 'GitHub',
+          position: 'right',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
```

Comment:
```suggestion
      style: 'light',
```

The dark footer is much too dark, it takes away a lot of focus on the main content because of this.
I also think the "light" footer looks friendlier than the big black footer.